tests/spec/fs.read.spec.js

```diff
@@ -88,3 +88,96 @@ describe('fs.read', function() {
     });
   });
 });
+
+describe('fs.promises.read', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fsPromises = util.fs().promises;
+    expect(fsPromises.read).to.be.a('function');
+  });
+
+  it('should return a promise', function() {
+    var fsPromises = util.fs().promises;
+    expect(fsPromises.read()).to.be.a('Promise');
+  });
+  it('should read data from a file', function() {
+    var fsPromises = util.fs().promises;
+    var wbuffer = new Filer.Buffer([1, 2, 3, 4, 5, 6, 7, 8]);
+    var rbuffer = new Filer.Buffer(wbuffer.length);
+    var fdesc;
+    rbuffer.fill(0);
+
+    return fsPromises.open('/myfile', 'w+')
+      .then((fd)=>{
+        fdesc=fd;
+        return fsPromises.write(fd, wbuffer, 0, wbuffer.length, 0);
+      })
+      .catch((error)=>{throw error;})
```

Comment:
A friendly suggestion: if you can remove this line because you don't need to catch error here.